assets/airports/cyhz.json

```diff
@@ -0,0 +1,657 @@
+        "XOMRO":    ["N44d42m93.00", "W063d40m38.00"],
+        "RW05":     ["N44d52m08.44", "W063d31m28.58"],
+        "URMIN":    ["N45d00m13.00", "W063d47m11.00"],
+        "XUBKO":    ["N45d03m84.00", "W063d19m77.00"],
+        "PUTUD":    ["N44d59m74.00", "W063d13m02.00"],
+        "NUBUK":    ["N44d57m85.00", "W063d36m30.00"],
+        "RW23":     ["N44d53m18.04", "W063d30m19.91"],
+        "PAULO":    ["N45d15m20.00", "W063d20m10.00"],
+        "NOTOP":    ["N45d27m16.00", "W062d00m72.00"],
+        "NUTBY":    ["N45d41m25.00", "W063d14m80.00"],
+        "CAAPE":    ["N45d18m00.00", "W065d17m77.00"],
+        "YZX":      ["N44d55m63.00", "W065d05m12.00"],
+        "YQI":      ["N43d49m51.00", "W066d04m95.00"],
+        "YQM":      ["N46d11m34.00", "W064d34m25.00"],
+
+        "BOSUB":    ["N45d11m83.00", "W062d55m52.00"],
+        "CAYLY":    ["N45d37m27.00", "W062d14m50.00"],
+        "KEBOT":    ["N44d59m18.00", "W063d22m63.00"],
+        "VIMLA":    ["N44d58m76.00", "W063d29m09.00"],
+        "MENVO":    ["N44d53m86.00", "W063d11m68.00"],
+        "KIXIT":    ["N44d48m33.00", "W063d08m31.00"],
+        "MUVOX":    ["N44d51m85.00", "W064d12m20.00"],
+        "EBONY":    ["N44d54m14.00", "W067d09m39.00"],
+        "EBKID":    ["N44d53m42.00", "W065d30m08.00"],
+        "ELERI":    ["N42d34m47.00", "W064d09m78.00"],
+        "ALLEX":    ["N44d25m00.00", "W067d00m00.00"],
+        "LIRLA":    ["N45d01m25.00", "W062d50m13.00"],
+        "BAFDY":    ["N44d46m66.00", "W064d46m95.00"],
+        "AGMIR":    ["N45d13m70.00", "W061d57m33.00"],
+        "ETGAR":    ["N44d25m16.00", "W064d09m78.00"],
+        "ILOTA":    ["N43d33m90.00", "W067d00m00.00"],
+        "TUSKY":    ["N43d33m90.00", "W067d00m00.00"],
+        "DUNUB":    ["N44d58m18.00", "W063d52m51.00"],
+        "RUDBU":    ["N44d51m87.00", "W063d23m11.00"],
+        "NOTUB":    ["N44d40m87.00", "W063d28m03.00"],
+        "EBLIT":    ["N44d40m17.00", "W063d16m42.00"],
+        "PEPTA":    ["N44d47m60.00", "W063d27m32.00"],
+        "OVTUV":    ["N44d59m31.00", "W063d15m70.00"]
+    },
+    "runways": [{
+        "name": ["05", "23"],
+        "end": [
+            ["N44d52m08.44", "W063d31m28.58", "449ft"],
+            ["N44d53m18.04", "W063d30m19.91", "443ft"]
+        ],
+        "ils": [false, true]
+    }, {
+        "name": ["14", "32"],
+        "end": [
+            ["N44d53m37.02", "W063d31m04.81", "435ft"],
+            ["N44d52m53.97", "W063d29m38.67", "476ft"]
+        ],
+        "ils": [true, false]
+    }],
+    "sids": {
+        "CYHZ4": {
+            "icao": "CYHZ4",
+            "name": "Halifax Four",
+            "rwy": {
+                "14": ["#053"],
+                "32": ["#323"],
+                "05": ["#053"],
+                "23": ["#233"]
+            },
+            "body": [],
+            "exitPoints": {
+                "CAAPE": ["ARDEE", "CAAPE"],
+                "VQI": ["HIDIG", "YQI"],
+                "NOTOP": ["IGTAS", "NOTOP"],
```

Comment:
So the question is whether or not there is a specific list of waypoints departing aircraft may use as their first fix. If there is, then we ought to consider them exitpoints of the vector SID. If they can go wherever they please, then we might have an issue because procedures with only a single waypoint (and a vector waypoint at that) might be an issue.